pom.xml

```diff
@@ -53,24 +50,19 @@
 				<!-- this plug-in has issues with username / author / committer / whatever property/name is null might be
 				see https://github.com/github/maven-plugins/issues/85
 				final solution was to configure user name on github account profile.
-				nevertheless we execute dryRun here, as the push to refs/heads/master will push to master of wiremock-extensions
-				repository instead of the specified mvn-repo (see repositoryName)
 				-->
 	            <configuration>
+	            	<repositoryOwner>9cookies</repositoryOwner>
 	            	<repositoryName>mvn-repo</repositoryName>
+	            	<server>github</server>
 	            	<branch>refs/heads/master</branch>
-	            	<outputDirectory>${repo.folder}/mvn-repo</outputDirectory>
+	            	<outputDirectory>${project.build.directory}/mvn-repo</outputDirectory>
+	            	<merge>true</merge>
 	                <message>Maven artifacts for ${project.artifactId}.${project.version}</message>
+	                <noJekyll>true</noJekyll>
```

Comment:
We should omit this since it creates an additional file in the repo - which we don't need.